### ISSUE TITLE
Subdivide into several repos

### ISSUE THREAD

**Issue by mbonsma** [phageParser/phageParser#260]
It makes sense at this point to split this monolithic repo into a few separate repos that don't have much overlap (suggested by @patrick-wu). 

Here are some possible divisions:

**phageParser/phageAPI**: repo for the Django app and RESTful API. I think that would be the [phageAPI](https://github.com/phageParser/phageParser/tree/master/phageAPI) folder and possibly [this](https://github.com/phageParser/phageParser/tree/master/restapi) too?

**phageParser/analysis** (or *dev* or something like that): for the bits of analysis scripts and new features that end up in the main directory, possibly also [parserscripts](https://github.com/phageParser/phageParser/tree/master/parserscripts) although maybe that belongs in the API repo? If any new repo were to retain the leftovers of the other moves, this should probably be it.

**phageParser/demo** (or something like that): just the [demos](https://github.com/phageParser/phageParser/tree/master/demos) folder and any new standalone Jupyter notebooks that only talk to the API. 

It seems like it may be possible to do these moves while preserving history: @patrick-wu found [this post](http://gbayer.com/development/moving-files-from-one-git-repository-to-another-preserving-history/) about it, and maybe @QuLogic you also know how to do thsi?

**Comment by patrick-wu** [comment]
I'll try the article's method sometime this weekend with a dummy repo. once we find a working method, we can get to work on migrating. 

here are my thoughts on the ones you listed:

**demo**: This one should be fairly straight forward and easy to move. We could probably start with this.
**phageAPI**: we should probably list out all the dependent files for the this one, because I think it's composed of 2 folders (the ones you linked) and also a few files in the root directory. We should also look through the code to see if it has dependencies in the parserscripts folder
**analysis**: I'm not super sure about this one yet, because we do still have the original phageParser repo. If we move out the API and the demos, the original repo can just be a place for us to have these scripts?

@mbonsma @cemyld

**Comment by mbonsma** [comment]
@patrick-wu I agree about the analysis one: it can be whatever's left over in the main repo after **demo** and **phageAPI** are moved out.

**Comment by patrick-wu** [comment]
@mbonsma the method worked (for the demos folder)! Can you invite me to the phageParser organization (not just the repo) so that I can create the new repository for the notebooks?

**Comment by mbonsma** [comment]
@patrick-wu done!
